test/src/tests-store.ts

```diff
@@ -522,7 +522,7 @@ describe('Store', () => {
                     ssd: 8675309,
                 } as IGlobalStoreV2MinifiedKeys,
                 cu: 'mpid3',
-                l: false,
+                l: true,
```

Comment:
Would this be 0 or 1 then?

Reply:
any particular reason to change this at all from `false` to `true`?  i think the update in this test would be to look for the `l` on the persistence.  seems to not be testing for it at all, and that's why this test is still passing.